.github/workflows/docker_build.yml

```diff
@@ -0,0 +1,49 @@
+name: Create and publish a Docker image
```

Comment:
the name of this one should be different than what's in main.yml

Reply:
Done.